x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/columns.tsx

```diff
@@ -344,11 +345,18 @@ export const getMonitoringColumns = (
     },
     {
       field: 'current_status.bulk_create_time_durations',
-      name: i18n.COLUMN_INDEXING_TIMES,
+      name: (
+        <EuiToolTip content={i18n.COLUMN_INDEXING_TIMES_TOOLTIP}>
+          <>
+            {i18n.COLUMN_INDEXING_TIMES}{' '}
+            <EuiIcon size="s" color="subdued" type="iInCircle" className="eui-alignTop" />
+          </>
+        </EuiToolTip>
+      ),
       render: (value: RuleStatus['current_status']['bulk_create_time_durations']) => (
         <EuiText data-test-subj="bulk_create_time_durations" size="s">
           {value != null && value.length > 0
-            ? Math.max(...value?.map((item) => Number.parseFloat(item)))
+            ? value.reduce<number>((prev, cur) => prev + Number(cur), 0).toFixed()
```

Comment:
Ended up removing decimal precision here, can add back desired `factionDigits` (2? 3?) if we think necessary. Only use case I could think of here is if users want to pivot to event-log from these metrics' exact values ([as written to event-log](https://github.com/elastic/kibana/pull/112286/files#diff-a299cd268d971c8eed401050e1acff88f797a1808e7f49f12f7b10e7ff32c0e8R65-R66)).